sdk/identity/azure-identity/tests/test_known_authority_hosts.py

```diff
@@ -0,0 +1,30 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+from azure.identity._internal.msal_credentials import MsalCredential
+
+
+def test_set_known_authority_host():
```

Comment:
With the new test case below, I think we can remove this old one which relies on MSAL's internal attribute. But it is up to you. I'm leaving an approval here, even before your fixing the build errors. :-)